assets/components/kube-apiserver/config-overrides.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: kubecontrolplane.config.openshift.io/v1
```

Comment:
All manifests under the `assets` directory will be applied once KAS notifies ready state. What are the implications of applying a new KAS configuration object afterwards?
In other control plane components, we create config files and store them in `/var/lib/microshift/` and pass them automatically. I'm not against this approach, I just want to understand the implications, and if we decide this is a better approach, we can move all configs to assets.

Reply:
This isn't a real type served by the API, it's only read from disk by OpenShift's patched kube-apiserver. Where are all manifests under assets being applied? I thought that individual asset paths had to explicitly listed for that to happen.